ansible_base/activitystream/models/entry.py

```diff
@@ -58,31 +58,19 @@ def changed_fk_fields(self):
 
         for op in ('added_fields', 'changed_fields', 'removed_fields'):
             for field_name, value in self.changes.get(op, {}).items():
-                field = self.content_type.model_class()._meta.get_field(field_name)
+                if (model := self.content_type.model_class()) is None:
+                    continue
+                field = model._meta.get_field(field_name)
                 if isinstance(field, models.ForeignKey):
-                    if op == 'changed_fields':
-                        pk = value[1]
-                    else:
-                        pk = value
-                    changed_fks[field_name] = pk
+                    try:
+                        fk_model = field.related_model
+                    except AttributeError:  # Likely the model was deleted
+                        continue
+                    pk = value[1] if op == 'changed_fields' else value
```

Comment:
Might be good to have a comment here...

So for _updates_ this implements a rule to show the _latter_ of the two values involved.